jupyterhub/notebook-images/overlays/cuda-11.0.3/gpu-notebook.yaml

```diff
@@ -42,7 +42,7 @@ items:
     output:
       to:
         kind: ImageStreamTag
-        name: 's2i-minimal-gpu-cuda-$(cuda_version)-notebook:latest'
+        name: 'minimal-gpu:cuda-$(cuda_version)'
```

Comment:
```
        name: 'minimal-gpu:cuda-$(cuda_version)-py3.8'
        or
        name: 'minimal-py3.8-gpu:cuda-$(cuda_version)'
```
should we define python version as well.

Reply:
@harshad16 I think the first one is good. Do we do the same thing for tf and pytorch?

Reply:
we can same thing for them too